src/components/Microsoft.DotNet.UpgradeAssistant.Analysis/AnalyzeResultWriter.cs

```diff
@@ -32,10 +33,26 @@ public async Task WriteAsync(IAsyncEnumerable<AnalyzeResultDefinition> results,
             {
                 Runs = await ExtractRunsAsync(results).ToListAsync(token).ConfigureAwait(false),
             };
+            if (format != null && format.ToLower(CultureInfo.CurrentCulture) == "html")
+            {
```

Comment:
use `is not null` instead of `!=null`. possibly use `string.Equals(format, "html", StringComparison.OrdinalIgnoreCase)`

Reply:
Done